src/main.ts

```diff
@@ -287,6 +287,19 @@ export default function main() {
 
     log(`feedback prompting enabled`);
 
+    if (sseChannel) {
```

Comment:
there's already a check here: https://github.com/bucketco/bucket-tracking-sdk/pull/75/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR267-R269

Reply:
True, but:
```
const res = await request(`${getUrl()}/feedback/prompting-init`, {
      userId,
    });
```
will split the function into 2+ promises which leads to a race condition. Since calling `user("alex"); user("john");` not being awaited, will both bypass that check.

Reply:
made the changes as discussed